docs/advanced/experimental/docker-compose.md

```diff
@@ -91,20 +90,18 @@ services:
     - 8080:8080
 ```
 
-Here you can see that 7 services will be created. Indeed, first each `Flow` has one entrypoint the `gateway`, then each
-`Deployment` of the encoder has one `Head` and two instances of the executors because it has two replicas. 
-It is the almost the same for the Indexer, excepts that the two instances are the two shards.
-
+Here you can see that 7 services will be created.One for the `gateway` which is the entrypoint of the `Flow`. 
+Then both of the Executors are wrapped into a `Deployment` .
```

Comment:
I think you do not need to talk about `Deployment`, just say that you see X services for each executors, corresponding to a Head, and the replicas ...